Source_Code/Common/Messages/ActionRequests/MoveRequest.java

```diff
@@ -3,21 +3,34 @@
 
 public class MoveRequest extends ActionRequest
 {
-   
+
+   public int PlayerID;
+   public Move moveDirection;
+
    /**
-    * Using a string to keep track of where the player wants to move. This seemed the
-    * most flexible until we figure out if we want to use room names to dictate 
-    * movement, or if we want to use directions.
+    * Enumerate valid moves for a player - these will be interpreted by 
+    * action request listener
     */
-   public String moveDirection;
-   
-   public int PlayerID;
-   
-   public MoveRequest( int PlayerID, String moveDirection )
+   enum Move

```

Comment:
This needs to be public to support our current ClueLessClient main (since we have MoveRequest sends in master)